app/src/main/java/com/duckduckgo/app/pixels/AppPixelName.kt

```diff
@@ -383,4 +383,9 @@ enum class AppPixelName(override val pixelName: String) : Pixel.PixelName {
     DEDICATED_WEBVIEW_URL_EXTRACTION_FAILED("m_dedicated_webview_url_extraction_failed"),
 
     BLOCKLIST_TDS_FAILURE("blocklist_experiment_tds_download_failure"),
+
+    SET_AS_DEFAULT_PROMPT_IMPRESSION("m_set-as-default_prompt_impression"),
```

Comment:
either use “-“ or “_” but don’t use a combination of both.

Reply:
I'm using this to treat the new pixels like a tree:
- `m`
  - `set-as-default`
    - `prompt`
      - `impression`
      - `click`
      - `dismissed`
    - `in-menu`
      - `click`

I see various examples of that in the codebase, but please let me know if that should cause any issues.